mysql-test/include/analyze-format.inc

```diff
@@ -0,0 +1,3 @@
+# The time on ANALYSE FORMAT=JSON is rather variable
+
+--replace_regex /("(r_total_time_ms|r_buffer_size)": )[^, \n]*/\1"REPLACED"/
```

Comment:
I have only one issue: .result files are used in discussions about what other `r_` parameters should be added, so it would be nicer if the field names remained intact.   Which should be easy to do as `replace_regex` supports multiple replaces, something like:
`
+--replace_regex /("(r_total_time_ms)": )[^, \n]*/\1"REPLACED"/   /("(r_buffer_size)": )[^, \n]*/\1"REPLACED"/ 
`